surefire-api/src/main/java/org/apache/maven/surefire/util/DefaultRunOrderCalculator.java

```diff
@@ -140,4 +203,10 @@ public int compare( Class o1, Class o2 )
             }
         };
     }
+
+    public ConsoleLogger getConsoleLogger()
```

Comment:
I'll change that logger like the TODO says; I just could not quickly find how they log stuff and printing to `System.out` makes the output ugly and may take time.